content/en/docs/setup/install/multicluster/before-you-begin/index.md

```diff
@@ -59,7 +59,7 @@ Istio may change slightly.
 
 This guide will assume that you use a common root to generate intermediate
 certificates for each cluster. Follow the [instructions](/docs/tasks/security/cert-management/plugin-ca-cert/)
-to generate and push a ca certificate secrets to both the `cluster1` and `cluster2`
+to generate and push a ca certificate secret to both the `cluster1` and `cluster2`
```

Comment:
```suggestion
to generate and push a CA certificate and secret to both the `cluster1` and `cluster2`
```

Reply:
Updated spelling but I think the key piece that needs to be in the cluster is the secret (which was made from the cert files). I'm understand that secret to be called the `CA certificate secret`.